src/garage/experiment/local_runner.py

```diff
@@ -60,7 +62,11 @@ def __init__(self, snapshot_config, max_cpus=1):
         self.plot = False
```

Comment:
while you're here, can you make all of these attrs `_private`? you can use @property if they need to be part of the API.